cloud/pulsar-client/README.md

```diff
@@ -4,9 +4,11 @@ The `pulsar-client` and `pulsar-perf` are CLI tool written in Java language for
 
 # Prerequisites
 
-- Pulsar broker 2.7.0-742fc5c9b+
+> Pulsar client is required to be newer than 2.6.1 which will include the OAuth2 authentication plugin.
 
-> You can get this tarball from [bintray](https://bintray.com/streamnative/maven/org.apache.pulsar/2.7.0-742fc5c9b). When Pulsar 2.6.1 is released, you can also use the official 2.6.1 version.
+- Pulsar broker 2.7.0-742fc5c9b+
+- Get the `SERVICE_URL` of your StreamNative Cloud Pulsar cluster: [How to get service URL](#How to get service URL)
```

Comment:
where is `#How to get service URL` and `#How to get token options` tracked?

Reply:
fix, PTAL again